community/broot/depends

```diff
@@ -0,0 +1,2 @@
+rust
```

Comment:
This should be marked as make dep

Reply:
Did I miss something or we are talking about comment marking it as such? There is no other mechanism to distinguish build and runtime deps, right?

Reply:
You'd change it to `rust make` to mark it as a make dep

Reply:
Wow, I genuinely thought that those _makes_ stand for make - the tool. 😮